Sources/ArcGISToolkit/Components/Compass/Compass.swift

```diff
@@ -110,10 +114,37 @@ public struct Compass: View {
                                  """
                     )
                 )
+                .snapToZeroSensoryFeedback(enabled: snapToZeroSensoryFeedbackEnabled, heading: heading)
         }
     }
 }
 
+private extension View {
+    /// Enables the snap to zero sensory feedback
+    /// when it is available.
+    @ViewBuilder
```

Comment:
We'll need to mark this unavailable for `visionOS` to not break the build.

```suggestion
    @available(visionOS, unavailable)
    @ViewBuilder
```